airflow/api_connexion/endpoints/dag_endpoint.py

```diff
@@ -43,11 +50,19 @@ def get_dag_details(dag_id):
     return dag_detail_schema.dump(dag)
 
 
-def get_dags():
+@provide_session
+def get_dags(session):
     """
     Get all DAGs.
     """
-    raise NotImplementedError("Not implemented yet.")
+    offset = request.args.get(parameters.page_offset, 0)
+    limit = min(int(request.args.get(parameters.page_limit, 100)), 100)
```

Comment:
I'm thinking that this would not allow us to have configurable maximum limit. What do you think?